src/util-mpm.h

```diff
@@ -68,7 +68,7 @@ typedef struct MpmPattern_ {
     uint8_t *original_pat;
     /* case sensitive */
     uint8_t *cs;
-    /* case INsensitive */
+    /* case Insensitive */
```

Comment:
I thought about this for a bit and have decided that the nonstandard casing didn't add anything of value...